planner/core/physical_plans.go

```diff
@@ -485,6 +485,9 @@ type PhysicalSort struct {
 // appear in final physical operator tree.
 type NominalSort struct {
 	basePhysicalPlan
+
+	ByItems    []*ByItems
+	OnlyColumn bool
```

Comment:
Please add some comments to explain why `NominalSort` would have these two fields.